.circleci/config.yml

```diff
@@ -41,8 +41,8 @@ jobs:
         # Cache all other dependencies and data
         - save_cache:
             paths:
-              - ~/circleci/repo/data
-              - ~/circleci/.linuxbrew
+              - ~/repo/data
```

Comment:
Looks good, I wasn't sure what the path would look like inside the circleci machine. Good to know for the future!